src/Vector.js

```diff
@@ -0,0 +1,78 @@
+/* Convenience class to store a vector and its associated flags */
+class Vector
+{
+	constructor(x, y)
+	{
+		this.components = [x, y];
+	}
+	
+	/**
+	 *  Creates one or more Vectors based on the string passed in.
+	 *  Returns an array of Vectors containing all Vectors described in the string.
+	 *
+	 *  If an invalidly formatted string is passed in, an error is logged to the
+	 *  console, but execution continues (and all validly formatted Vectors are
+	 *  output normally).
+	 */
+	static Create(str)
```

Comment:
Note my earlier question about the `Piece.Create` function. Moreover, I think there is a better name for this method that points to the fact that it returns an array of `Vector`.